packages/bsky/src/services/indexing/plugins/gate.ts

```diff
@@ -0,0 +1,95 @@
+import { AtUri } from '@atproto/syntax'
```

Comment:
Awkwardly I found we don't really make use of the new `gate` index—we always grab the gate record and just work with it that way.  We can do this mostly because the gate uri can be derived from the post uri.  But it also feels a little odd not to index them at all.

Reply:
yeah i think we just index for now - there are other reasons that it's useful to have this table around - for instance introspecting it ourselves. we may find we need it down the line as well